acceptance/tests/environment/negative/agent_run_should_fail_if_env_unreadable.rb

```diff
@@ -1,5 +1,7 @@
 test_name "C97899 - Agent run should fail if environment is unreadable" do
   skip_test 'requires a puppetserver master for managing the environment' if hosts_with_role(hosts, 'master').length == 0 or not @options[:is_puppetserver]
+  jruby_version = on(master, 'puppetserver ruby --version').stdout
+  skip_test 'This is only valid on JRuby 1.7' if jruby_version =~ /^jruby 9/
```

Comment:
Would this be more future proof if it was restricted to the 1.x series rather than 9? That way is won't fail again when/if JRuby bumps to 10.x or 3000.

```
skip_test unless jruby_version =~ /^jruby 1.7/
```